app/experimenter/experiments/models/nimbus.py

```diff
@@ -82,6 +82,31 @@ def __str__(self):
         return self.name
 
 
+class NimbusBranch(models.Model):
+    experiment = models.ForeignKey(
+        NimbusExperiment,
+        blank=False,
+        null=False,
+        related_name="branches",
+        on_delete=models.CASCADE,
+    )
+    is_control = models.BooleanField(default=False)
+    enabled = models.BooleanField(default=True)
+    name = models.CharField(max_length=255, blank=False, null=False)
+    slug = models.SlugField(max_length=255, blank=False, null=False)
+    description = models.TextField(default="")
+    ratio = models.PositiveIntegerField(default=1)
```

Comment:
Ratio's never include decimal places? Are there docs somewhere on what a Nimbus branch is?

Reply:
This is in terms of the DTO but describes the requirements for ratio : https://docs.google.com/document/d/1jZQ--ozuQ0_Su-lh7-dX-3yqmWsIan2QPYyTt6lv4ow/edit#heading=h.5bv7czvcn6dm